### DISCUSSION TITLE
Cannot find any documentation on how to use this library.

### DISCUSSION THREAD

**Discussion by Rsverma** [augustoproiete/microsoft-solver-foundation-express-nuget#9] [Q&A] [answered]
Please find a link to any reference of the API  or documentation regarding getting started with the code.

**Comment by augustoproiete** [comment, marked as answer]
@Rsverma The official docs from Microsoft are on [docs/Microsoft Solver Foundation 3.1](https://docs.microsoft.com/en-us/previous-versions/visualstudio/ff524509(v=vs.93)?redirectedfrom=MSDN). That might be the best starting point.

There are many other posts and articles you can find via search engines, which might be helpful too:

- [First steps with the Microsoft Solver Foundation](https://brandewinder.com/2011/07/16/First-steps-with-the-Microsoft-Solver-Foundation/)
- [Solving optimization problems with Microsoft Solver Foundation](https://www.codeproject.com/Articles/1183168/Solving-optimization-problems-with-Microsoft-Solve)
- [Modeling with Solver Foundation 3.0](https://channel9.msdn.com/posts/Modeling-with-Solver-Foundation-30)
- [Microsoft Solver Foundation Quick Start](https://jamesmccaffrey.wordpress.com/2014/04/13/microsoft-solver-foundation-quick-start/)
